plugin-localization/src/main/java/com/mapbox/mapboxsdk/plugins/localization/MapLocale.java

```diff
@@ -348,6 +348,7 @@
     LOCALE_SET.put(new Locale("pt", "PT"), PORTUGAL);
     LOCALE_SET.put(new Locale("pt", "BR"), BRAZIL);
     LOCALE_SET.put(new Locale("vi", "VN"), VIETNAM);
+    LOCALE_SET.put(new Locale("vi"), VIETNAM);
```

Comment:
How did you test it? Can we somehow make sure it indeed fixes an issue fully?

Reply:
A customer already tested it with the snapshot.